### ISSUE TITLE
Problem when total = buffered

### ISSUE THREAD

**Issue by jaumard** [suragch/audio_video_progress_bar#26]
In my case I'm streaming a web radio, so there is no total at all as it never ends. So to paliate this I'm doing this:

```dart
ProgressBar(
              progress: playerDuration.position,
              buffered: playerDuration.buffered,
              total: playerDuration.buffered,
              onSeek: (duration) {
                print('User selected a new time: $duration');
              },
            ),
```

But by doing so when audio is paused and stream is still buffering, then the thumb circle position is not updated which give this weird thing:

<img width="366" alt="Capture d’écran 2021-10-16 à 16 04 22" src="https://user-images.githubusercontent.com/2970449/137590475-4aa68111-6282-48dc-83a5-79eac2352f82.png">

Basically if total changes, the thumb position should be updated accordingly.

Thanks a lot for this great package!

**Comment by suragch** [comment]
If you are still using this package, try out version 0.10.0. There is a fix in there from @addie9000 that may solve your issue.

**Comment by suragch** [comment]
I'm going to close this for now, but feel free to reopen if it is still a problem.
